lib/tasks/db_tasks.rake

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'active_record'
+require 'active_support'
+
+namespace :db do
+  namespace :migrate do
+    desc "Migrates the database by adding partition_guid and encryption_epoch to specified array of models (must specify :models)"
+    task :add_encryption_fields, [:models] => :environment do |_, args|
+      models = args[:models]
+
+      abort 'Need to provide `models` as argument' if models.blank?
+
+      models.each do |model|
+
+        model_classname = model.to_s.camelize.singularize.constantize
+        model_tablename = model.to_s.tr('/', '_')
+
+        add_column model_tablename, :partition_guid, :string
+        add_column model_tablename, :encryption_epoch, :datetime
+
+        model_classname.reset_column_information
+        model_classname.find_each do |record|
+          record.generate_partition_guid
+          record.generate_encryption_epoch
+
+          record.save!(validate: false)
+        end
+
+        change_column model_tablename, :partition_guid, :string, null: false
+        change_column model_tablename, :encryption_epoch, :datetime, null: false
+      end
+    end
+
+    desc "Migrates the database by adding the encryption_keys table"
+    task :add_encryption_keys_table do
```

Comment:
We also need to add the unique index for guid and for partition_guid+epoch

Reply:
This does not seem straightforward to do either with a rake task, esp with the multi column index.  I think for this part, at least for now, I'll also just include instructions in the readme to add the indexing.

Reply:
Discussed at planning. To move this forward agreed to use README instructions for the migration instead of rake task.